Update users/me endpoint docs (#3404)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/reference/api/users/index.md

| :------------ | :------------------------------------------------------------------- | :----------- | :------------------------------------------ | :------- |
| profile       | Updated profile for user                                             | Body         | [Profile object](#profile-object)           | FALSE    |

> End user can only update `profile` with this request. Within the profile, if the end user tries to update the primary or the secondary email IDs, verification emails will be sent to those email IDs and the fields will get updated only upon verification. To update credentials, use [Update Profile with ID](#update-profile-with-id).

Reviewer: ```suggestion
End user can only update `profile` with this request. Within the profile, if the end user tries to update the primary or the secondary email IDs, verification emails are sent to those email IDs, and the fields are updated only upon verification. To update credentials, use [Update Profile with ID](#update-profile-with-id).
```
